ui/app/templates/components/identity/item-metadata.hbs

```diff
@@ -1,4 +1,5 @@
-{{#each-in model.metadata as |key value|}}
+{{log "model" model this.model @model}}
```

Comment:
Do we need to log this or just forgot to remove?

Reply:
Just forgot! Thanks